### PR TITLE
CE-3095: Filter other type on classification

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
ping @idradm @sqreek

### CODE REVIEW

File: service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/classification/UserOverrideClassification.java [hunk 1]
```diff
@@ -13,8 +13,9 @@
   @Override
   public String classify(TemplateClassificationData classificationData) {
     return classificationData.getProviders().stream()
-        // filter providers containing more than one type
-        .filter(provider -> provider.getTypes().size() == 1)
+        // filter providers containing more than one type or type other
+        .filter(provider -> provider.getTypes().size() == 1
+                            && !provider.getTypes().contains(OTHER_TYPE))
```
Comment (idradm) [inline service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/classification/UserOverrideClassification.java]: You can add separate filter method for that, should look nicer

**Comment by idradm** [comment]
What about unit tests for logic? Could you add a case for what you are changing?

File: service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/classification/UserOverrideClassification.java (hunk 1 above)
Comment (sqreek) [inline service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/classification/UserOverrideClassification.java]: :+1:

**Comment by idradm** [comment]
awesomely awesome!

**Pull request merged**